source/Halibut.Tests/Transport/ConnectionManagerFixture.cs

```diff
@@ -22,7 +22,7 @@ public async Task DisposedConnectionsAreRemovedFromActive_WhenMultipleConnection
         {
             var connectionFactory = CreateFactoryThatCreatesTestConnections();
 
-            var serviceEndpoint = new ServiceEndPoint("https://localhost:42", Certificates.TentacleListeningPublicThumbprint);
+            var serviceEndpoint = new ServiceEndPoint("https://localhost:42", Certificates.TentacleListeningPublicThumbprint, new HalibutTimeoutsAndLimitsForTestsBuilder().Build());
```

Comment:
Nit: For later maybe, but I wonder if this would be easier with making a ServiceEndPointBuilder? 🤔